src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1524,7 +1524,7 @@ public int Compare(Symbol fst, Symbol snd)
                     node = node.Parent;
                 }
 
-                CSDiagnosticInfo info = NotFound(where, simpleName, arity, where.ToString(), diagnostics, aliasOpt, qualifierOpt, options);
+                CSDiagnosticInfo info = NotFound(where, simpleName, arity, simpleName, diagnostics, aliasOpt, qualifierOpt, options);
```

Comment:
It looks like there are only two call sites for this function and now both are passing the same value for the second and the forth parameter. We should probably get rid of one of them to simplify things.